docusaurus/docs/dev-docs/quick-start.md

```diff
@@ -29,34 +27,53 @@ We will first create a new Strapi project on your machine by running a command i
 Follow the steps below by clicking on the togglable content to read more instructions.
 
 <details style={detailsStyle}>
-<summary style={summaryStyle}>Step 1: Run the installation script</summary>
+<summary style={summaryStyle}>Step 1: Run the installation script and create a Strapi Cloud account</summary>
 
-### Step 1: Run the installation script
+### Step 1: Run the installation script and create a Strapi Cloud account
 
-Run the following command in a terminal:
+1. Run the following command in a terminal:
 
-<Tabs groupId="yarn-npm">
+    <Tabs groupId="yarn-npm">
 
-<TabItem value="yarn" label="Yarn">
+    <TabItem value="yarn" label="Yarn">
 
-```bash
-yarn create strapi-app my-project --quickstart
-```
+    ```bash
+    yarn create strapi-app my-strapi-project --quickstart
+    ```
 
-</TabItem>
+    </TabItem>
 
-<TabItem value="npm" label="NPM">
+    <TabItem value="npm" label="NPM">
 
-```bash
-npx create-strapi-app@latest my-project --quickstart
-```
+    ```bash
+    npx create-strapi-app@latest my-strapi-project --quickstart
+    ```
+
+    </TabItem>
+
+    </Tabs>
+
+2. The terminal will invite you to create a Strapi Cloud account and start a free, 14-day trial. Ensure `Login/Sign up` is selected in the terminal, or use arrow keys to select it, and press Enter.
+
+3. In the new browser tab that opens, ensure the confirmation code is the same as in the terminal and click **Confirm**.<br/>Alternatively, if a browser tab did not automatically open, copy and paste the link from the terminal and type the confirmation code into the form if it's not pre-filled already, then click **Confirm**.
+
+4. Still in the browser tab, click **Continue with GitHub**. If you are not already logged in into GitHub with your current browser session, you might be redirected to a GitHub login page. 
 
-</TabItem>
+5. Once logged in, the browser will display a "Congratulations, you're all set!" message and you can safely close the browser tab and get back to the terminal.
 
-</Tabs>
+<ThemedImage
+  alt="Login GIF"
+  sources={{
+    light: '/img/assets/quick-start-guide/qsg-cloud-login.gif',
+    dark: '/img/assets/quick-start-guide/qsg-cloud-login.gif',
+  }}
+/>
+
+As you will see in the terminal, your project is now building locally and also getting ready to be deployed on Strapi Cloud later.
 
 :::info
-The `quick start` installation sets up Strapi with a SQLite database. Other databases and installation options are available (see [CLI installation guide](/dev-docs/installation/cli)).
+* The `quick start` installation sets up Strapi with a SQLite database. Other databases and installation options are available (see [CLI installation guide](/dev-docs/installation/cli)).
+* The folder of your project will include a `.strapi-cloud.json` file used to link the local Strapi project on your machine to the Strapi Cloud project.
```

Comment:
The `.strapi-cloud.json` file is only here if the user has selected "Login/signup" while creating his strapi project or tried to do a `strapi deploy` in the project's folder.

Reply:
Yes, and that's what we're teaching them to do here 😅 The point of the Quick Start Guide is not to cover all options but to explain what happens when they do what we told them to do :D